openarena/README.md

```diff
@@ -22,11 +22,12 @@ docker build -t ${ACR}/openarena:${TAG} .
 docker push ${ACR}/openarena:${TAG}
 ```
 You can run the above script on [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/wsl2-index).
+
 5. Create a new MPS Build either via playfab.com or via [the CreateBuildWithCustomerContainer API call](https://docs.microsoft.com/en-gb/rest/api/playfab/multiplayer/multiplayerserver/createbuildwithcustomcontainer?view=playfab-rest). On this Build, select Linux VMs, the image:tag container image you uploaded and a single port for the game, 27960/UDP. 
 6. Wait for the Build to be deployed
 7. To allocate a server and get IP/port, you can use the [RequestMultiplayerServer](https://docs.microsoft.com/en-gb/rest/api/playfab/multiplayer/multiplayerserver/requestmultiplayerserver?view=playfab-rest) API call. For more information you can check the [documentation](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers)
 8. Download [OpenArena client](https://openarena.fandom.com/wiki/Main_Page), open the game executable for your platform and connect to your server. Enjoy!
 
 ### Is this the recommended way I should use GSDK?
 
-Definitely not. Standard output/error stream processing is hacky at its best. For best results, you should use the [proper GSDK for your language of choice](https://github.com/PlayFab/gsdk).
\ No newline at end of file
+Definitely not. Standard output/error stream processing is hacky at its best. For best results, you should use the [proper GSDK for your language of choice](https://github.com/PlayFab/gsdk).
```

Comment:
I didn't change anything here... I'm not sure what Github thinks the difference is

Reply:
Ah. Opened in CodeFlow and now I see that there's an extra blank line at the end of the file now (must have been auto-inserted for me)

---
In reply to: [524709381](https://github.com/PlayFab/gsdkSamples/pull/23#discussion_r524709381) [](ancestors = 524709381)